### ISSUE TITLE
supervisor: log runnable restarts into their own log DNs

### ISSUE THREAD

**Issue by lorenz** [monogon-dev/monogon#296]
Currently the supervisor logs runnable restarts into its own `supervisor` DN. This hides runnable restarts from DN-filtered logs, potentially confusing people who look at the logs as a lot of our runnables do not log errors, instead relying on the supervisor to log them as the reason for restarting the runnable. It would be a much better experience if these were inline with the actual runnable logs. For example #295 can only really be understood by looking at both the original DN logs as well as the supervisor's.

**Comment by q3k** [comment]
Yeah, I've been meaning to fix that for a while. We should just change this behaviour.
